integration-tests/http/__tests__/order/admin/order.spec.ts

```diff
@@ -66,6 +66,69 @@ medusaIntegrationTestRunner({
         expect(response.data.orders).toHaveLength(0)
```

Comment:
Q: I feel like you have some issues with the linting rules. I would have expected the format to be a bit different :) could you have a look at it please